### PR TITLE
IPInstance API and related changes

### PR DESCRIPTION
**Pull request by mars1024**
Pull Request Description
---

### Special notes for reviews

### CODE REVIEW

File: pkg/apis/networking/v1/conversion.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+func parseIPInstanceVersion(ipInstance *IPInstance) string {
+	if len(ipInstance.Labels[constants.LabelVersion]) == 0 {
+		return IPInstanceV11
+	}
+	return ipInstance.Labels[constants.LabelVersion]
+}
+
+func CanonicalizeIPInstance(cluster cluster.Cluster) (err error) {
+	var c client.Client
+	if c, err = client.New(cluster.GetConfig(), client.Options{
```
Comment (hhyasdf) [inline pkg/apis/networking/v1/conversion.go:47]: Why not use an exist cached client of manager?

File: pkg/apis/networking/v1/conversion.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+		return err
+	}
+
+	for i := range ipInstanceList.Items {
+		var ipInstance = &ipInstanceList.Items[i]
+		if parseIPInstanceVersion(ipInstance) == IPInstanceLatestVersion {
+			continue
+		}
+
+		if err = convertIPInstanceToLatestVersion(ipInstance, getPodUID); err != nil {
```
Comment (hhyasdf) [inline pkg/apis/networking/v1/conversion.go:81]: Has the conversion be updated to apiserver?

Comment (mars1024) [reply pkg/apis/networking/v1/conversion.go:76]: fixed

File: pkg/apis/networking/v1/conversion.go (hunk 1 above)
Comment (mars1024) [reply pkg/apis/networking/v1/conversion.go]: fixed

File: pkg/apis/networking/v1/conversion.go (hunk 2 above)
Comment (mars1024) [reply pkg/apis/networking/v1/conversion.go:76]: fixed

**Review by hhyasdf — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**